src/routes/channel.js

```diff
@@ -0,0 +1,81 @@
+const express = require('express');
+const router = express.Router();
+const { check, validationResult } = require('express-validator');
+const passport = require('passport');
+const jwtAuth = passport.authenticate('jwt', { session: false });
```

Comment:
You could probably remove this and pass the `jwtAuth` in the `app.js` file because we want to protect all channel endpoints.